web/styles/popovers.css

```diff
@@ -82,6 +82,7 @@
 }
 
 .tippy-box[data-theme="popover-menu"] {
+    overflow: hidden;
```

Comment:
What is the impact of this change on other popovers? 

@sayamsamal can you review this detail? We might need this scoped to just typeahead.

Reply:
We definitely don't want this here, as the tippy arrow is placed with absolute positioning outside the tippy box, and this would hide the arrow.

Reply:
Also, I checked and the typeahead uses "dropdown-widget" tippy theme, so this has no effect on the typeahead.

Reply:
This was to fix an issue mentioned in https://github.com/zulip/zulip/pull/30141#issuecomment-2146202718.
It no longer exists so removing this change.